domene/src/main/java/no/nav/foreldrepenger/selvbetjening/oppslag/domain/Søkerinfo.java

```diff
@@ -6,7 +6,9 @@
 
 import com.fasterxml.jackson.annotation.JsonInclude;
 
+import jakarta.validation.constraints.NotNull;
+
 @JsonInclude(NON_NULL)
-public record Søkerinfo(PersonFrontend søker, List<Arbeidsforhold> arbeidsforhold) {
+public record Søkerinfo(@NotNull PersonFrontend søker, @NotNull List<Arbeidsforhold> arbeidsforhold) {
```

Comment:
Aller objekter på ha @Valid på seg for at validering skal propagere inn i klassen.

@Valid @NotNull PersonFrontend søker

Reply:
Hva vil @Size ha å si hvis man ikke spesifiserer min eller max?